src/backends.jl

```diff
@@ -135,7 +135,7 @@ function pickDefaultBackend()
             Pkg.installed(env_default)  # this will error if not installed
             sym = Symbol(lowercase(env_default))
             if haskey(_backendType, sym)
-                return backend(sym)
+                return @eval backend($sym)
```

Comment:
Hm, I do not see that `You have set PLOTS_DEFAULT_BACKEND=GR but it is not installed.` warning anywhere -- neither in Juno, IJulia nor the REPL.
And I'm also not seeing any world age issues anymore, but that might just be because of the Atom.jl PR that makes use of `invokelatest`. Or did that appear for anyone not using Juno?